src/sdam/common.ts

```diff
@@ -36,7 +37,8 @@ export const ServerType = Object.freeze({
   RSArbiter: 'RSArbiter',
   RSOther: 'RSOther',
   RSGhost: 'RSGhost',
-  Unknown: 'Unknown'
+  Unknown: 'Unknown',
+  LoadBalancer: 'LoadBalancer'
```

Comment:
Are these names fixed and can we rely on them? I didn’t realize it so far, but since we make decisions based on the ServerType/TopologyType in mongosh, we’ll need to update that to account for the new values

Reply:
Yes, per the spec the `TopologyType` is always `"LoadBalanced"` and the `ServerType` is always `"LoadBalancer"`

Reply:
https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#topologytype